### ISSUE TITLE
[A11y] Desktop/Mobile Explore Collection - Grid: All Image Thumbs

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1796]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Explore Collections

### Admin user?

- [ ] Yes

### Issue Description

Explore Collection - Grid: https://staging.oregondigital.org/explore_collections/all?locale=en&view=masonry

For all the 8 images displayed in the grid, the following instance failed, see below.

Failed Instance:
image-alt: Ensures <img> elements have alternate text or a role of none or presentation. 

Path:
.masonry.document.collection:nth-child(1) > .thumbnail > a > img

Snippet
`<img src="/assets/collection-a38b932554788aa578debf2319e8c4ba8a7db06b3ba57ecda1391a548a4b6e0a.png">`

### Suggested Fix

Fix ONE of the following:
Element does not have an alt attribute
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by CGillen** [comment]
Fixed by #2037
